content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -404,6 +406,71 @@ instances:
 ### Serverless environment
 
 For more information about AWS Lambda functions, see [how to connect your Lambda telemetry using tags][15].
+
+### OpenTelemetry
+
+When using OpenTelemetry, map the following [resource attributes][16] to their corresponding Datadog conventions:
+
+| OpenTelemetry convention | Datadog convention |
+| --- | --- |
+| `deployment.environment` | `env` |
+| `service.name` | `service` |
+| `service.version` | `version` |
+
+<div class="alert alert-warning">Do not use Datadog-specific tags like <code>DD_SERVICE</code>, <code>DD_ENV</code> or <code>DD_VERSION</code> in your OpenTelemetry configuration.</div>
+
+{{< tabs >}}
+{{% tab "Environment variables" %}}
+
+To set resource attributes using environment variables, set `OTEL_RESOURCE_ATTRIBUTES` with the appropriate values:
+
+```shell
+export OTEL_RESOURCE_ATTRIBUTES="service.name=my-service,deployment.environment=production,service.version=1.2.3"
+```
+
+{{% /tab %}}
+
+{{% tab "SDK" %}}
+
+To set resource attributes in your application code, create a `Resource` with the desired attributes and associate it with your `TracerProvider`.
+
+Here's an example using Python:
+
+```python
+from opentelemetry.sdk.resources import Resource
+from opentelemetry.sdk.trace import TracerProvider
+
+resource = Resource(attributes={
+   "service.name": "<SERVICE>",
+   "deployment.environment": "<ENV>",
+   "service.version": "<VERSION>"
```

Comment:
OpenTelemetry Python has a package that defines these constants: https://opentelemetry.io/docs/languages/python/instrumentation/#add-semantic-attributes

We can make use of it here; we would have to import:
```python
from opentelemetry.semconv.resource import ResourceAttributes
```

and then we can use `ResourceAttributes.SERVICE_NAME`, `ResourceAttributes.DEPLOYMENT_ENVIRONMENT` and `ResourceAttributes.SERVICE_VERSION` here

Reply:
Ah, good note! Since we only have this one code example for now, I wonder if it'd be better to leave the simple example with string literals instead of these constants?

This might be more easily understood for those less familiar with OpenTelemetry, and it'd translate to other languages more easily?

Reply:
That's fine, yes, the main benefit here is if the semantic conventions name change, but these three are not going to change any time soon, so we can just use the strings there